include/lcp-parameters.php

```diff
@@ -85,7 +85,11 @@ public function get_query_params($params){
     }
 
     if ( $this->utils->lcp_not_empty('before') ) {
-      $this->before = $params['before'];
+      if(strcmp(strtolower($params['before']), "today") == 0) {
```

Comment:
Could you change line 88 to:
```php
if( 'today' === strtolower($params['before']) {
```
?

There is no reason to use `strcmp` when you only want to check if two strings are exactly the same. Also note that `strcmp` executes longer than `===`.

https://stackoverflow.com/questions/3333353/string-comparison-using-vs-strcmp

Reply:
Hi @zymeth25 
That's a good point. I've fixed line 88.

Reply:
Ah, sorry, I forgot a closing paren!!!

Reply:
Fixed my review now.

Reply:
Oh, I didn't pay attention to it either 😄

Reply:
Awesome, thanks for the edit.